### ISSUE TITLE
Question about grapheme set

### ISSUE THREAD

**Issue by kkp15** [as-ideas/DeepPhonemizer#14]
Hello. Thank you for this amazing repository! 
I have a question though. What’s the easiest way to get a unique grapheme set for a specific language? How did you get that list when training a multilingual model?

**Comment by cschaefer26** [comment]
Hi, you can just extract it from the training data. E.g. you collect the set of characters from it and then paste the result into the config. That's basically how i proceeded for the trained models (I filtered some graphemes though).

**Comment by skanda1005** [comment]
Hi @cschaefer26 , I wanted to train the model for hindi, but had doubts on how I need to make the config file, especially the input and output because I'm getting index out of range error. Thanks!

**Comment by cschaefer26** [comment]
Hi, you can use the standard config file, but you will have to adjust the text_symbols and phoneme_symbols according to the symbols that occur in your dataset!

**Comment by skanda1005** [comment]
Got it working, thanks!

**Comment by cschaefer26** [comment]
Nice, let me know if you run into issues.

**Comment by skanda1005** [comment]
Hi, so I realized that in my phoneme set, if some of the phonemes have multiple characters, it doesn't get parsed and those multiple char phones are either removed or replaced after preprocessing.
Any solutions to this issue?

**Comment by cschaefer26** [comment]
Hi, multiple characters shouldn't be a problem, the cmudict model has multi-char phonemes: https://github.com/as-ideas/DeepPhonemizer#:~:text=en_us_cmudict_forward

You can pass each sample as a tuple of [str, list, list], e.g. ('en', ['foo'], ['bar'])

**Comment by skanda1005** [comment]
So, I am training it in hindi and phones like t͡ʃ and ẽː dont get parsed. I used these as inputs for the tokenizer and there is no output meaning it doesn't get tokenized.
PS. t͡ʃ is actually 3 chars, not 2. Would that cause a problem?

**Comment by cschaefer26** [comment]
No that should be fine. Actually your example looks more like there should be just three phoneme chars as output (t͡ʃ). Just make sure the symbols are present in the config (phoneme_symbols, e.g.  '͡')

**Comment by skanda1005** [comment]
Oh, So should I separate the chars of that phone as 3 different elements in the list? 
e.g ['t', '͡', 'ʃ')

**Comment by cschaefer26** [comment]
Yes, that's also how the standard config is set. You can then simply provide the phonemized words as strings.

**Comment by skanda1005** [comment]
Great, one last question. I have around 11k entries in my dataset and modified the config file for val_counts to be 1000. So I train it on around 10k entries. Is there any way I can test out my model on the val_dataset and compare it to the ground truth?
Thanks for all your help!
